addons/web_editor/static/lib/odoo-editor/src/OdooEditor.js

```diff
@@ -2707,6 +2707,15 @@ export class OdooEditor extends EventTarget {
         }
         return !!selectionInBlockRoot;
     }
+    /**
+     * Returns true if the provided node can suport html content
+     *
+     * @param {Node} node
+     * @returns {boolean}
+     */
+    nodeSupportHtmlContent(node) {
+        return !closestElement(node, '[data-oe-model]:not([data-oe-field="arch"]),[data-oe-translation-id]', true);
+    }
```

Comment:
This function does not reference `this` and therefore should be an util outside of OdooEditor.js IMO.

Reply:
:+1:  I'll move this in utils